operator/src/main/java/org/keycloak/operator/controllers/KeycloakDeployment.java

```diff
@@ -241,8 +241,7 @@ private StatefulSet createBaseDeployment() {
 
         if (!containerBuilder.hasReadinessProbe()) {
             containerBuilder.withNewReadinessProbe()
-                .withInitialDelaySeconds(20)
-                .withPeriodSeconds(2)
+                .withPeriodSeconds(1)
```

Comment:
I'd make these changes in a separate issue in order to have it more tracked and not hidden in some issues related to the testsuite.

AFAIK, it also means the combination of the period seconds and failure threshold concludes in cutting the failure time to half. 

The container is considered not ready:
Before changes: 2s\*250 = 500+20 = 520s = 8.5min
After changes: 1s\*250 = 250s = ~4min

I believe it's correct, but I don't see any problem to lower that. 
WDYT?

Reply:
> WDYT?

Agreed on the effect of changing the periodseconds, I had put something in the initial pr comment on that.  

For the failureThreshold / total time, I'm unsure as to why it was as high as it is - the recent mentioning of long start up times from reading a db in another az gave me pause about what can be expected.  Since it seems like we're open to something like this change, I'll separate off a pr with a cross-reference to #21111 to discuss further.

Reply:
Moved to #21884